### PR TITLE
refactor(SectionHeader): rid off absolute position

### LINKED ISSUE #361: Get rid off absolute position in the section header
The position of BackLink should not be absolute https://github.com/sendsay-ru/account/blob/webpack-react/client/components/SectionHeader/SectionHeaderBacklink.css#L10; we should add a modificator to the section title instead.

### PR DESCRIPTION
**Pull request by zouxuoz**
resolve #361

### CODE REVIEW

File: client/components/SectionHeader/SectionHeader.js [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
+    </div>
+  );
+};
 
 SectionHeader.propTypes = {
   children: PropTypes.node.isRequired,
   block: PropTypes.func.isRequired,
   element: PropTypes.func.isRequired,
+  title: PropTypes.string.isRequired,
+  backlinkLabel: PropTypes.string.isRequired,
```
Comment (radiansz) [inline client/components/SectionHeader/SectionHeader.js]: I guess both backlink params shouldnt be required,

File: client/components/SectionHeader/SectionHeader.js [hunk 2]
```diff
@@ -1,17 +1,39 @@
...
+        {backlink}
+        <div {...element('title')}>{title}</div>
+        {children}
+      </div>
+    </div>
+  );
+};
 
 SectionHeader.propTypes = {
   children: PropTypes.node.isRequired,
```
Comment (radiansz) [inline client/components/SectionHeader/SectionHeader.js]: Children shouldnt be required

File: client/routes/subscribers/components/SubscribersHeader.js [hunk 3]
```diff
@@ -42,9 +42,7 @@ const messages = defineMessages({
 
 const SubscribersHeader = ({ intl, subscribersCount }) => (
   <header>
-    <SectionHeader>
-      <SectionHeaderTitle text={subscribersCount.fetching ? 'Loading...' : `${intl.formatMessage(messages.title)} ${subscribersCount.data}`} />
-    </SectionHeader>
+    <SectionHeader title={subscribersCount.fetching ? 'Loading...' : `${intl.formatMessage(messages.title)} ${subscribersCount.data}`} />
```
Comment (radiansz) [inline client/routes/subscribers/components/SubscribersHeader.js]: With current realization it's hard to add something additional to main title, like number in subscribers pages or tag of inactive form in forms pages. Also it's tricky to add SectionActions which is here almost everywhere. I would propose something like this:
'''
<SectionHeader>
  <SectionTitle title=... backlink=... backlinkto=... />
  [<SectionActions>]
</SectionHeader>
'''

**Comment by radiansz** [comment]
LGTM

**Comment by radiansz** [comment]
lgtm

**Comment by rosendi** [comment]
lgtm

**Comment by radiansz** [comment]
lgtm

**Pull request merged**